File: PHP/classi/Polizza.php
<?php

class Polizza
{
    private $id;
    private $id_viaggio;
    private $tipologiaMerce;
    private $peso;
    private $fornitore;
    private $giorniMagazzinaggio;
    private $tariffa;

    public function __construct($id, $id_viaggio, $tipologiaMerce, $peso, $fornitore, $giorniMagazzinaggio, $tariffa)
    {
        $this->id = $id;
        $this->id_viaggio = $id_viaggio;
        $this->tipologiaMerce = $tipologiaMerce;
        $this->peso = $peso;
        $this->fornitore = $fornitore;
        $this->giorniMagazzinaggio = $giorniMagazzinaggio;
        $this->tariffa = $tariffa;
    }

    public function getId()
    {
        return $this->id;
    }

    public function getIdViaggio()
    {
        return $this->id_viaggio;
    }

    public function getTipologiaMerce()
    {
        return $this->tipologiaMerce;
    }

    public function getPeso()
    {
        return $this->peso;
    }

    public function getFornitore()
    {
        return $this->fornitore;
    }

    public function getGiorniMagazzinaggio()
    {
        return $this->giorniMagazzinaggio;
    }

    public function getTariffa()
    {
        return $this->tariffa;
    }
}

File: SPRINGBOOT/demo/src/main/java/com/example/classes/Polizza.java
package com.example.classes;

public class Polizza {
    private int id;
    private int id_viaggio;
    private String tipologiaMerce;
    private double peso;
    private String fornitore;
    private int giorniMagazzinaggio;
    private double tariffa;

    public Polizza(int id, int id_viaggio, String tipologiaMerce, double peso, String fornitore, int giorniMagazzinaggio, double tariffa) {
        this.id = id;
        this.id_viaggio = id_viaggio;
        this.tipologiaMerce = tipologiaMerce;
        this.peso = peso;
        this.fornitore = fornitore;
        this.giorniMagazzinaggio = giorniMagazzinaggio;
        this.tariffa = tariffa;
    }

    public int getId() {
        return id;
    }

    public int getId_viaggio() {
        return id_viaggio;
    }

    public String getTipologiaMerce() {
        return tipologiaMerce;
    }

    public double getPeso() {
        return peso;
    }

    public String getFornitore() {
        return fornitore;
    }

    public int getGiorniMagazzinaggio() {
        return giorniMagazzinaggio;
    }

    public double getTariffa() {
        return tariffa;
    }
}
